### ISSUE TITLE
This code acc != source paper acc

### ISSUE THREAD

**Issue by busishu** [ganeshjawahar/mem_absa#14]
Thx your code. But when i use this code in restaurant dataset, i run 100 epochs and got 70% acc.
Here is my hyper-parameters:
'batch_size': 1,
 'edim': 300,
 'init_hid': 0.1,
 'init_lr': 0.01,
 'init_std': 0.01,
 'lindim': 100,
 'max_grad_norm': 100,
 'mem_size': 83,
 'nepoch': 320,
 'nhop': 9,
 'nwords': 3066,
 'pad_idx': 0,
 'pretrain_embeddings': 'glove.840B.300d.txt',
 'show': True, 

I know something is wrong, but i cant fix it. 
For example, increase batch_size but i got less than 20% accuracy. And i change the number of hops but nothing happen.

**Comment by amanchuan** [comment]
Hi, busishu. my accuracy is also less than 20%.... can you understand the meaning of "lindim" parameter? I don't understand the what's meaning of "linear part of the state".
